config/settings.yml

```diff
@@ -26,7 +26,7 @@ redis_cache_url:
 redis_cache_password:
 redis_cache_ssl: false
 slack_webhook: 'this is set through a env var in Openshift and not shared for security reasons'
-disable_rbac: 'false'
+disable_rbac: false
```

Comment:
Same here, do not do changes into this file as this is something related to production.